test/fixtures/typescript-function/src/index.ts

```diff
@@ -0,0 +1,5 @@
+import { ReferenceId } from "sf-fx-sdk-nodejs"
+
+console.log("Here is code!");
```

Comment:
Could you make this file be an example hello world function, representative of what a customer might be using?

Reply:
Sounds good. I frankly don't know what one looks like. I was trying to go with the maximum separation of concerns. I'm waiting on Lillian to publish the template and I'll modify that so it can run with little/no context and no salesforce access needed.

Reply:
Updated. I also really like your suggestion of having circle run the tests in the project. I moved the fixture to a top level "fixtures" directory. This helps avoid future mocha collisions from having nested test files under "test". I also had it run linting and found two issues with the source template code.